### ISSUE TITLE
Reduced galactocerebrosidase activity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7626]
**Preferred term label:**
Diminished enzyme activity of galactocerebrosidase.
PMID: 34015306
Comment: Reduced galactocerebrosidase activity is a feature of Krabbe disease.

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
245200 
PMID: 20886637  29/29

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
